angles/test/utest.cpp

```diff
@@ -160,7 +195,7 @@ TEST(Angles, normalize_angle)
  EXPECT_NEAR(M_PI/2, normalize_angle(5*M_PI/2), epsilon);
  EXPECT_NEAR(M_PI/2, normalize_angle(9*M_PI/2), epsilon);
  EXPECT_NEAR(M_PI/2, normalize_angle(-3*M_PI/2), epsilon);
- 
+
```

Comment:
We try not to make unnecessary whitespace only changes if possible to avoid future merge conflicts.

Reply:
Sorry, I did not notice this change. I will restore it back to the original state.